docker/api

```diff
@@ -0,0 +1,11 @@
+FROM ruby
```

Comment:
I'm probably ok with binding to latest in this case, mainly because I plan to run integration tests against this.  Any breaking change won't be accepted into master and thus not released.